### DISCUSSION TITLE
commands to return only the metadata

### DISCUSSION THREAD

**Discussion by Exafit** [crunchy-labs/crunchy-cli#343] [Ideas]
Would that be an idea or is it already possible to receive only the metadata as the return of a command?

I was thinking that I could build the command into a script to fully automate the download. Then I just have to pass a txt with URLs and it creates appropriate folders and subfolders for the series and seasons. However, this would require the respective name of the series.
At the end, it should be possible to check whether all files have been successfully downloaded or whether files are missing based on the existing files and the information on how many episodes there are (or based on the episode names).
You could also push the current status via ntfy.sh or other services, but this also requires metadata.

**Comment by Exafit** [comment]
Ah ok, I overlooked the search command. 

But it seems that this one has a bug. 
When I try to display the metadata for Naruto or Naruto Shippuden, it just says: 
":: √ Logged in" and nothing else.
Other series work, i.e. Attack on Titan. 
I haven't yet found a reason as to which ones work and which don't

**Comment by Frooastside** [comment]
You can simply use the `--output` cli argument to create folders like this

```bash
crunchy-cli archive -o "{series_name}/Season {season_number}/{title} S{season_number}E{episode_number}.mkv" <URL>
```
this will create a folder with series name, a subfolder with the seasons and put the episodes in the appropriate folders.


If you want to get metadata you can use the [search command](https://github.com/crunchy-labs/crunchy-cli?tab=readme-ov-file#search)
